vitest.config.ts

```diff
@@ -9,6 +9,9 @@ export default defineConfig({
   },
   plugins: [svelte({ hot: !process.env.VITEST })],
   test: {
+    outputDiffMaxLines: Infinity,
+    outputDiffMaxSize: 1000,
+    outputTruncateLength: Infinity,
```

Comment:
Can you elaborate on these changes?

Reply:
I can remove them, was trying to get the output of a ttest failure to show up